packages/manager/apps/pci-kubernetes/src/components/cart/Cart.component.tsx

```diff
@@ -0,0 +1,72 @@
+import { JSX, useMemo } from 'react';
+
+import { useTranslation } from 'react-i18next';
+
+import { Button, Icon, Text } from '@ovhcloud/ods-react';
+
+import {
+  Cart as BaseCart,
```

Comment:
Question: Then why not having the Cart from /cart/components called 'BaseCart' or 'CartLayout' ? Or change the name of this one

It will be very confusing if I want to use the 'Cart' component when there are two in /cart and /cart.component.